app/lib/nomis_client/court_cases.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module NomisClient
+  class CourtCases < NomisClient::Base
+    class << self
+      ACTIVE_CASES_FILTER = 'activeOnly=true'
+
+      def get(booking_id)
+        court_cases_route = "/bookings/#{booking_id}/court-cases?#{ACTIVE_CASES_FILTER}"
+
+        begin
+          NomisClient::Base.get(court_cases_route).body
+        rescue StandardError => e
+          raise e
+        end
```

Comment:
I think you could remove this exception handler as it's not really doing anything

Reply:
yes, this kind of "reminder" to handle errors from nomis, but will handle them in a different ticket  (we need a general solution)